src/main/java/org/kainos/ea/DropwizardTheITCrowdServiceConfiguration.java

```diff
@@ -0,0 +1,10 @@
+package org.kainos.ea;
+
+import io.dropwizard.Configuration;
```

Comment:
This is the only import that is used here, the reset can be removed

Reply:
Done